tests/smoke/base_deploy_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package smoke
+
+import (
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
+	logger = capnslog.NewPackageLogger("github.com/rook/rook", "integrationTest")
+
+	defaultNamespace     = "default"
+	defaultRookNamespace = "rook"
```

Comment:
could you make the default namespace something completely different like `testrook`? Then the default case for coverage is guaranteed to catch different defaults from what all the devs are testing.